test/sqllogictest/attributes/mir_unique_keys.slt

```diff
@@ -78,15 +78,16 @@ INNER JOIN
 ON t.b = u.d;
 ----
 Explained Query:
-  Project (#1, #0, #2, #0) // { keys: "([1])" }
-    Join on=(#0 = #3) type=differential // { keys: "([0])" }
+  Project (#1, #0, #0, #0) // { keys: "([1])" }
+    Join on=(#0 = #2) type=differential // { keys: "([0])" }
       ArrangeBy keys=[[#0]] // { keys: "([0])" }
         Reduce group_by=[#1] aggregates=[sum(#0)] // { keys: "([0])" }
           Filter (#1) IS NOT NULL // { keys: "()" }
             ReadStorage materialize.public.t // { keys: "()" }
-      ArrangeBy keys=[[#1]] // { keys: "([0], [1])" }
-        Filter (#0 = #1) // { keys: "([0], [1])" }
-          ReadIndex on=v v_primary_idx=[*** full scan ***] // { keys: "([0, 1])" }
+      ArrangeBy keys=[[#0]] // { keys: "([0])" }
+        Project (#1) // { keys: "([0])" }
```

Comment:
Uncertain why this `Project` uses `#1` instead of `#0`. Maybe harmless, but unexpected.

Reply:
Short answer: this `Project` does not exist when `EquivalencePropagation` is run. Some other transform is introducing it.